### PR TITLE
Adds import command to contentpacks

### PR DESCRIPTION
**Pull request by rtibbles**
Takes the import functionality from KA Lite contentload and implements it within content pack maker.

### CODE REVIEW

File: contentpacks/__main__.py [hunk 1]
```diff
@@ -3,32 +3,37 @@
...
+    generate_kalite_language_pack_metadata, translate_assessment_item_text, Catalog
+from contentpacks.import_channel import retrieve_import_data
 import logging
 
-def make_language_pack(lang, version, sublangargs, filename, no_assessment_items, no_subtitles, no_assessment_resources):
-    node_data, subtitle_data, interface_catalog, content_catalog = retrieve_language_resources(version, sublangargs, no_subtitles)
+
+def make_language_pack(lang, version, sublangargs, filename, no_assessment_items, no_subtitles,
+                       no_assessment_resources):
+    node_data, subtitle_data, interface_catalog, content_catalog = retrieve_language_resources(version, sublangargs,
```
Comment (aronasorman) [inline contentpacks/__main__.py:38]: You linter!

Comment (rtibbles) [inline contentpacks/__main__.py:38]: I had done some editing on the server in nano - the linting was needed.

File: contentpacks/import_channel.py [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    "Exercise": ["exercise"],
+}
+
+file_kind_map = {}
+
+for key, value in file_kind_dictionary.items():
+    for extension in value:
+        file_kind_map[extension] = key
+
+file_meta_data_map = {
```
Comment (aronasorman) [inline contentpacks/import_channel.py:59]: Can you add an explanation as to what this constant is for?

Comment (rtibbles) [inline contentpacks/import_channel.py:59]: Sure!

File: contentpacks/import_channel.py [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+        })
+
+        if kind != "Exercise":
+            node.update({
+                "format": extension,
+            })
+            # Copy over content
+            content_dir = os.path.join(os.path.join(os.getcwd(), "build", "content"))
+            if not os.path.exists(content_dir):
+                os.mkdir(content_dir)
```
Comment (aronasorman) [inline contentpacks/import_channel.py:206]: change to:

```
os.makedirs(content_dir, exist_ok=True)
```

To force the creation of parent directories as well.

**Comment by aronasorman** [comment]
It generated a zip file. Attempting to import to KA Lite.

**Comment by benjaoming** [comment]
@rtibbles - this seems like a lot of work was made for this PR, would it be worth importing it or is it no longer relevant?

**Comment by rtibbles** [comment]
The code was used for the Aflatoun deployment, the only reason we might want to use the code again is if we wanted to quickly bootstrap an Aflatoun Kolibri channel. If we do, we can revisit it :)

**Comment by aronasorman** [comment]
Stale. ka-lite-content-packs is deprecated now in favor of Studio's sushi chefs.

**Pull request closed without merging**